### ISSUE TITLE
Auto generation of dartlang issue due to an exception leaks user information (directories).

### ISSUE THREAD

**Issue by apwilson** [dart-atom/dartlang#795]
It'd be nice if we could strip out the folder paths before the .atom folder in the stack traces.

**Comment by apwilson** [comment]
This is true of the sdk location as well.  Is that really important to a bug?

**Comment by devoncarew** [comment]
It does leak PII. Unfortunately the exception handling and reporting is done in core Atom. It would be good if it sanitized paths, and had an option to pass config settings or not (or let users preview and edit them).
